configs/config.clevo_ns70pu

```diff
@@ -9,6 +9,8 @@ CONFIG_CBFS_SIZE=0xA00000
 CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="NS5x_NS7xPU"
 # CONFIG_USE_PM_ACPI_TIMER is not set
 CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x20000
+CONFIG_EDK2_BOOT_TIMEOUT=4
+CONFIG_USE_LEGACY_8254_TIMER=y
```

Comment:
Unneeded, causes sleep issues and increased power draw. Please remove

Reply:
Removed.